src/main/java/ca/corefacility/bioinformatics/irida/config/security/IridaWebSecurityConfig.java

```diff
@@ -89,6 +89,7 @@ protected void configure(HttpSecurity http) throws Exception {
 				.antMatchers("/dist/**").permitAll()
 				.antMatchers("/v3/api-docs/**").permitAll()
 				.antMatchers("/password_reset/**").permitAll()
+				.antMatchers("/ajax/password_reset/**").permitAll()
```

Comment:
please remove the ** at the end as that will allow unauthenticated access to the admin function to reset a users password. Which will then return a server error instead of unauthorized error.

Reply:
From our discussion we can keep this as is, and instead just move `adminNewPasswordReset` into `UsersAjaxController`.

Reply:
Updated in 8eb8bbe2496fefa283364aa24502b68c69fb49aa